generic/interbench.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+# Based on code by Brandon Philips
+#   copyright: 2006 IBM
+#   https://github.com/autotest/autotest-client-tests/tree/master/interbench
+
+import os
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Interbench(Test):
+
+    """
+    Interbench is designed to measure the effect of changes in Linux kernel
+    design or system configuration changes such as cpu, I/O scheduler and
+    filesystem changes and options. With careful benchmarking, different
+    hardware can be compared
+    """
+
+    def setUp(self):
+        '''
+        Build interbench
+        Source:
+        http://ck.kolivas.org/apps/interbench/interbench-0.31.tar.bz2
+        '''
+        sm_manager = SoftwareManager()
+        if not sm_manager.check_installed("gcc") and not sm.install("gcc"):
```

Comment:
oups, too late, this value should have been `sm_manager`. Never mind, fixed it in https://github.com/avocado-framework/avocado-misc-tests/commit/304be8856086600cf3323fbd8dd8bba3212fd622